ymer/src/main/java/com/avanza/ymer/support/CurrencyWriteConverter.java

```diff
@@ -15,11 +15,20 @@
  */
 package com.avanza.ymer.support;
 
+import java.util.Currency;
+
 import org.springframework.core.convert.converter.Converter;
 import org.springframework.data.convert.WritingConverter;
 
-import java.util.Currency;
-
+/**
+ * Converter that converts from a {@link Currency} to a {@link String}.
+ *
+ * @deprecated
+ * The converter {@code CurrencyToStringConverter}
+ * already exist in Spring ({@code MongoConverters}).
```

Comment:
Maybe these could be @link to these classes?

Reply:
You're right, javadoc does not support linking to package-private classes (as they're not supposed to be accessed)... So I guess using @code is the best way to refer to this classes, just as you did 👍